cli/sql.go

```diff
@@ -74,11 +75,47 @@ http://www.cockroachlabs.com/docs/
 `)
 }
 
+// handleSet handles the \set command and changes client-side flags.
+func handleSet(cmd []string, confirm *bool) int {
+	if len(cmd) != 2 {
+		fmt.Fprintf(os.Stderr, "Usage:\n \\set <option>\n")
+		return cliNextLine
+	}
+
+	switch cmd[1] {
+	case `confirm`:
+		*confirm = true
+	case `noconfirm`:
```

Comment:
I'd rather use a syntax like `\set confirm=false`; this will scale better to non-boolean settings.